io/driver/module_unload_load.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
+DRIVER=`lspci -k | grep -iw "Kernel driver in use" | cut -d ':' -f2 | sort | uniq`
+
+#Loads the kernel driver modules
+module_load() {
+    echo "Reloading driver $1"
+    modprobe $1
+    if [ $? != 0 ]; then
+        echo "Failed to load driver module $1"
+        break;
+    fi
+    echo
+}
+
+#Unloads the kernel driver modules
+module_unload() {
+    for i in $(cat $CONFIG_FILE | grep "$1=" | awk -F'=' '{print $2}'); do
+        module_unload $i
+        if [ $? != 0 ]; then
+            echo "Failed to unload driver module $i"
+            return
+        fi
+    done
+    echo "Unloaded driver $1"
+    rmmod $1
```

Comment:
The purpose is to unload and load all the kernel loaded modules for 100 times. 
Earlier, the code did not have the functionality of what to do when the module is not found. It now has been changed to handle this

Using "-e " is not feasible here. When used with -e, the code exits if a module is not found and this is not the expected behaviour. The code needs to skip the execution when the module is not found and continue with the rest of them.